capplets/windows/window-properties.ui

```diff
@@ -0,0 +1,684 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.1 -->
+<interface>
+  <requires lib="gtk+" version="3.20"/>
```

Comment:
`mate-control-center` requires gtk 3.22

Reply:
@sc0w What version of glade do you use?

![Captura de pantalla a 2019-12-22 14-07-49](https://user-images.githubusercontent.com/10171411/71322144-9f1eb080-24c4-11ea-9b31-16c7e5c0dbf8.png)

Reply:
I didn't open the file with glade, I just reviewed the code

Glade have this issue, already fixed, reported by me:
https://gitlab.gnome.org/GNOME/glade/issues/289

Reply:
OK, the UI file was verified with https://github.com/GNOME/glade/commit/213447c582623555f7bda571a2d9132087e27979.